pkg/arvo/ted/dns/address.hoon

```diff
@@ -30,12 +30,12 @@
   ?.  good
     %+  strand-fail:strandio  %bail-early-self-check
     [>"couldn't access ship on port 80"< ~]
+  ;<  our=@p  bind:m  get-our:strandio
+  ;<  ~       bind:m  (watch:strandio /sub collector-app /(scot %p our))
```

Comment:
```suggestion
  ;<  ~       bind:m  (watch:strandio /(scot %p our) collector-app /(scot %p our))
```

Reply:
it subscribes on a different wire than it expects in take-fact so you never get the response